requirements/requirements.in

```diff
@@ -1,6 +1,7 @@
 aiohttp
 ansible-runner>=1.4.6
 ansiconv==1.0.0  # UPGRADE BLOCKER: from 2013, consider replacing instead of upgrading
+autobahn==20.12.3  # CVE-2020-35678
```

Comment:
We have instructions to upgrade secondary dependencies without having to add it to this file.

But it does get added to this file, it seems like `>=` would be more correct than `==`. While there was a CVE, it was fixed, and it's really on us, due to the fact that we maintain `requirements.txt` in the first place.

Reply:
Yea, `>=` is probably fine here.